VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/select-hearing-layout.component.spec.ts

```diff
@@ -36,6 +37,17 @@ describe('SelectHearingLayoutComponent', () => {
         expect(component.selectedLayout).toBe(layout);
     });
 
+    it('should call translate service to update hearing layout option when language changes', () => {
+        component.currentText = textButton.innerHTML.toLowerCase().split(' ').join('-');
```

Comment:
I think you should setup your own values for this test rather than pulling from textButton.

Reply:
yea sure I can do that